build/systemd_units/manageiq-cockpit_ws@.service

```
@@ -0,0 +1,11 @@
+[Unit]
+PartOf=cockpit_ws.target
+[Install]
+WantedBy=cockpit_ws.target
+[Service]
+WorkingDirectory=/var/www/miq/vmdb
+Environment=BUNDLER_GROUPS=manageiq_default,ui_dependencies
+ExecStart=/bin/bash -lc 'exec ruby lib/workers/bin/run_single_worker.rb MiqCockpitWsWorker --heartbeat --guid=%i'
```

Didn't get the evm environment completely into properties files, so can we go with this for now?
It is much more non-root friendly

```suggestion
ExecStart=/bin/bash -c '. /etc/default/evm && exec ruby lib/workers/bin/run_single_worker.rb MiqCockpitWsWorker --heartbeat --guid=%i'
```

---

:+1: I'm good with that

The biggest thing holding this up is the rpm_build PR: https://github.com/ManageIQ/manageiq-rpm_build/pull/134

---

Actually I'm going to hold off on this, the point of this change is to statically define what we currently write out dynamically.

We can make the change you're talking about independently of this change, either before or after it is merged.  If you want you can update the unit file template in `MiqWorker#systemd_common` and I'll re-generate these, or we can make that change to the service files after this is merged.